Expensify/App
# Fix: Tooltip and message date position

### Details

1. Style changes to make the Tooltip responsive instead of fix width.
2. Apply the Tooltip containerStyle in index.native.js for consistency across platforms
3. Fixes the message timestamp to the same line as the displayName

### Fixed Issues

Fixes [an issue with the below PR ](https://github.com/Expensify/Expensify.cash/pull/3830#issuecomment-872556352) where the Tooltip and the Timestamp of a message would me misplaced.
$ PR #3830 

### Tests

1. Open any chat
2. Check that the timestamp is line with the displayName
3. Hoover over the displayName
4. The Tooltip should show close to your cursor


### QA Steps

Same as above.
### Tested On


### Screenshots


#### Web

https://user-images.githubusercontent.com/53711423/125451787-291abf84-df6d-4de2-a7e2-4b577aef5220.mov


#### Mobile Web


https://user-images.githubusercontent.com/53711423/125451822-1b42ce10-4169-49e3-a811-f310c0b4bb3b.mp4



#### Desktop


https://user-images.githubusercontent.com/53711423/125451839-824bee7e-cab9-4be4-a334-840f298ffbdc.mov




#### iOS


https://user-images.githubusercontent.com/53711423/125451846-375d8db5-0714-4352-9fa7-cab7fb327130.mp4




#### Android


https://user-images.githubusercontent.com/53711423/125451860-1cf24716-b4d6-40f4-8f86-d09898cb0b87.mp4
